### ISSUE TITLE
Visual Studio 2022 Support

### ISSUE THREAD

**Issue by scottkuhl** [madskristensen/HtmlTools2019#6]
### Installed product versions
- Visual Studio: 2022 All Versions

### Description
Make the extension available in Visual Studio 2022.

### Steps to recreate
1. Download the extension installer.
2. Run the installer.

### Current behavior
Message that it is already installed to all applicable products appears even though it is not installed in Visual Studio 2022.
It is also not listed in Online Extensions in Manage Extensions Visual Studio 2022.

### Expected behavior
Installs and works in Visual Studio 2022.

_Note: Sorry for the all the extra details.  I probably could have stopped with the title on this one._

**Comment by bairog** [comment]
@madskristensen What is estimated timeframe for porting this perfect extention for Visual Studio 2022?

**Comment by madskristensen** [comment]
It won't be. There is no compatible APIs for this particular extensions in VS 2022

**Comment by bairog** [comment]
> It won't be. There is no compatible APIs for this particular extensions in VS 2022

Ohhh, sad to hear it. 
Do you mean that VS Team removed some APIs that were present in VS 2019? Is it a matter of discussion or a final decision?
How do they motivate it? Is there an official replacement for the capabilities of this extension in the VS 2022 itself?
Thank you in advance.

**Comment by madskristensen** [comment]
Yes, they removed the unofficial and undocumented APIs required for this extension to work.
